css/style.css

```diff
@@ -0,0 +1,4 @@
+p {
+    color: blue;
+}
+
```

Comment:
Please consider to leave 2 spaces as indentation in your style.css. To have a good clean code.

Reply:
You gave a polite suggestion and explained the reason you suggested the change 👍 .